test/com/gfredericks/test/chuck/generators_test.clj

```diff
@@ -86,3 +87,13 @@
 (defspec subsequence-spec 100
   (prop/for-all [[xs ys] subsequence-gen]
     (subsequence? xs ys)))
+
+(def sub-map-gen
+  (gen'/for [m (gen/map gen/string-alphanumeric gen/nat)
+             sm (gen'/sub-map (gen/return m))]
+    [m sm]))
+
+(defspec sub-map-spec 100
+  (prop/for-all [[m sm] sub-map-gen]
+    (subsequence? (set (keys sm))
+                  (set (keys m)))))
```

Comment:
I don't think `subsequence?` is appropriate here, since maps are unordered. Probably `clojure.set/subset?`, or even `(every? #(= (find m (key %)) %) sm)` to check the values as well.